providers/terraform-provider-csbsqlserver/connector/connector_suite_test.go

```diff
@@ -0,0 +1,80 @@
+package connector_test
+
+import (
+	"database/sql"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/cloudfoundry/csb-brokerpak-aws/terraform-provider-csbsqlserver/connector"
+	"github.com/cloudfoundry/csb-brokerpak-aws/terraform-provider-csbsqlserver/testhelpers"
+)
+
+func TestConnector(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Database Connector Suite")
+}
+
+var (
+	port          int
+	adminPassword string
+	db            *sql.DB
+	conn          *connector.Connector
+)
+
+var _ = BeforeSuite(func() {
+	adminPassword = testhelpers.RandomPassword()
+	port = testhelpers.FreePort()
+	session := testhelpers.StartServer(adminPassword, port)
```

Comment:
This StartServer set up creates a database. So strictly speaking we are only testing that we can connect and create bindings when the database is already there. We are missing testing the scenario when the database has not yet been created.

Reply:
Discussed with @zucchinidev and agreed to provide the tests in a separate pull request to unblock other tasks. The scanario missing is tested with acceptance tests as well and those pass, so we have confidence this works.